### ISSUE TITLE
bug about training on 16khz speech

### ISSUE THREAD

**Issue by UkiTenzai** [redmist328/APCodec16_speech_infer#1]
Based on https://github.com/[YangAi520/APCodec](https://github.com/YangAi520/APCodec), I conducted training on the 16kHz LibriTTS train-clean-100 dataset. However, the training results were quite poor, and the performance metrics are far behind those of the pretrained model provided here. The generated audio lacks even basic formant structure, and the loss curve also behaves abnormally.

Do you have any suggestions or possible solutions to improve the training outcome?

![Image](https://github.com/user-attachments/assets/0e80014e-80af-46ef-a21b-f63c879cfc8c)
